src/commonTest/kotlin/exchange.dydx.abacus/validation/TradeRequiredInputTests.kt

```diff
@@ -213,12 +213,7 @@ class TradeRequiredInputTests : V3BaseTests() {
                         "trade": {
                             "type": "STOP_MARKET"
                         },
-                        "errors": [
-                            {
-                                "type": "WARNING",
-                                "code": "STOP_MARKET_ORDER_MAY_NOT_EXECUTE"
-                            }
-                        ]
+                        "errors": null
```

Comment:
[q] is this supposed to be an unhappy path test? if so should we change the test input params to ensure this still throws an error?

Reply:
eh it wasn't really an unhappy path - it was just a warning (non blocking for a user; errors are blocking) that we ripped out. I think I'm fine leaving it like this for now; we could add a test for `SELL_TRIGGER_TOO_CLOSE_TO_LIQUIDATION_PRICE` but I didn't touch that logic (and the mock data doesn't currently have a liquidation price) so inclined to leave it like this for now!